src/collections/_documentation/platforms/javascript/angular.md

```diff
@@ -16,23 +16,81 @@ yarn add @sentry/browser
 npm install @sentry/browser
 ```
 
+Then initialize new Sentry instance and configure Angular with `ErrorHandler` provided and explained below:
+
 ```typescript
 import { BrowserModule } from "@angular/platform-browser";
 import { NgModule, ErrorHandler, Injectable } from "@angular/core";
+import { HttpErrorResponse } from "@angular/common/http";
 
+import { environment } from "../environments/environment";
 import { AppComponent } from "./app.component";
 
 import * as Sentry from "@sentry/browser";
 
 Sentry.init({
-  dsn: "___PUBLIC_DSN___"
+  dsn: "___PUBLIC_DSN___",
+  // TryCatch has to be configured to disable XMLHttpRequest wrapping, as we are going to handle
+  // http module exceptions manually in Angular's ErrorHandler and we don't want it to capture same error twice.
+  // Please note that TryCatch configuration requires at least @sentry/browser v5.16.0.
+  integrations: [new Sentry.Integrations.TryCatch({
+    XMLHttpRequest: false,
+  })],
 });
 
 @Injectable()
 export class SentryErrorHandler implements ErrorHandler {
   constructor() {}
+
+  extractError(error) {
+    // Try to unwrap zone.js error.
+    // https://github.com/angular/angular/blob/master/packages/core/src/util/errors.ts
+    if (error && error.ngOriginalError) {
+      error = error.ngOriginalError;
+    }
+
+    // We can handle messages and Error objects directly.
+    if (typeof error === "string" || error instanceof Error) {
+      return error;
+    }
+
+    // If it's http module error, extract as much information from it as we can.
+    if (error instanceof HttpErrorResponse) {
```

Comment:
This is great. Have helped people do this piecemeal from time to time, but so much better to have it documented with a tested solution. 🎉